### PR TITLE
Release Manager role, schedule for next releases

### PR DESCRIPTION
**Pull request by jpkrohling**
Dear @jaegertracing/jaeger-maintainers, I would like to make a governance proposal, introducing the maintainer duties section. The only duty is to participate in releasing Jaeger every now and then, as specified in the changes to the release instructions file.

I would like to request a review from all maintainers.

﻿Signed-off-by: Juraci Paixão Kröhling <juraci@kroehling.de>

### CODE REVIEW

File: RELEASE.md
Comment (pavolloffay) [inline RELEASE.md:29]: We should clarify what components are mandatory or optional to release. 

It's fine to document this in the release.md https://github.com/jaegertracing/jaeger/blob/master/RELEASE.md. 
The doc already mentions publishing of docs, but it does not force releasing the UI.

**Review by pavolloffay — DISMISSED** [review]

File: RELEASE.md
Comment (jpkrohling) [reply RELEASE.md:29]: This *is* the release.md you linked :-) The case you have in mind is what I thought when I wrote this:

> This person will coordinate the required changes and will address any problems that might happen during the release

Required changes here might be in the other repositories, like IDL/lib/UI. Given that it wasn't clear enough, I just updated the paragraph to spell it out.

File: RELEASE.md
Comment (joe-elliott) [inline RELEASE.md:37]: Is it important to list all past release managers? Is this mainly just meant to give credit to the person doing the work? 

Feels like just one more thing to do during release :)

Comment (jpkrohling) [reply RELEASE.md:37]: Great point. I originally just wanted to share the past data via Slack, to justify the order for the future releases but thought it could be a good idea to have it listed somewhere. I'm fine with removing it.

File: GOVERNANCE.md
Comment (yurishkuro) [inline GOVERNANCE.md:40]: "Only one duty" is not accurate, maintainers are expected to review PRs or otherwise participate.

File: RELEASE.md
Comment (yurishkuro) [inline RELEASE.md:70]: Dates would be useful here.

File: RELEASE.md
Comment (pavolloffay) [reply RELEASE.md:37]: I agree with Joe, no need to list historical data. Going forward the table will get updated with future releases. It won't be necessarily done during the release

**Comment by jpkrohling** [comment]
PR updated to include the tentative release dates and to remove the past releases. Here are the release managers for past versions:

| Version   | Release Manager  |
|-----------|------------------|
| 1.8.0     | @pavolloffay     |
| 1.8.1     | @pavolloffay     |
| 1.8.2     | @pavolloffay     |
| 1.9.0     | @pavolloffay     |
| 1.10.0    | @pavolloffay     |
| 1.10.1    | @black-adder     |
| 1.11.0    | @pavolloffay     |
| 1.12.0    | @pavolloffay     |
| 1.13.0    | @pavolloffay     |
| 1.13.1    | @pavolloffay     |
| 1.14.0    | @pavolloffay     |
| 1.15.0    | @yurishkuro      |
| 1.15.1    | @yurishkuro      |
| 1.16.0    | @pavolloffay     |
| 1.17.0    | @yurishkuro      |
| 1.17.1    | @pavolloffay     |
| 1.18.0    | @pavolloffay     |
| 1.18.1    | @jpkrohling      |
| 1.19.0    | @objectiser      |
| 1.19.1    | @yurishkuro      |
| 1.19.2    | @objectiser      |
| 1.20.0    | @objectiser      |
| 1.21.0    | @joe-elliott     |
| 1.22.0    | @albertteoh      |
| 1.23.0    | @jpkrohling      |

File: RELEASE.md
Comment (jpkrohling) [reply RELEASE.md]: Added.

File: GOVERNANCE.md
Comment (jpkrohling) [reply GOVERNANCE.md]: Added a paragraph before that to reflect that. My original intent was to say that the only concrete task we require them to do is release. All others are things we want maintainers to do, but we don't actively ask them to, except perhaps for a few PRs where we tag people specifically based on their knowledge of the affected code.

Anyway, hopefully it's better now.

File: RELEASE.md
Comment (jpkrohling) [reply RELEASE.md]: Alright, I removed it.

**Review by yurishkuro — DISMISSED** [review]

**Review by joe-elliott — APPROVED** [review]

**Review by albertteoh — APPROVED** [review]

File: GOVERNANCE.md
Comment (pavolloffay) [inline GOVERNANCE.md:42]: > concrete

_93216643_concretepourindistock-157317173

**Review by pavolloffay — APPROVED** [review]

**Review by vprithvi — APPROVED** [review]

**Comment by jpkrohling** [comment]
We have approvals from all current maintainers, this is good to be merged.

**Pull request merged**
